temperature.js

```diff
@@ -0,0 +1,14 @@
+// passes js standard test: https://github.com/feross/standard
+const Temperature = {
+  toFahrenheit: (celsius) => {
```

Comment:
Could also be written as
```js
  toFahrenheit(celsius) {
```
If you are not concerned about losing lexical `this`